tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/create_expected_ct_list_test.py

```diff
@@ -115,11 +119,21 @@ def setUp(self):
         """).render(project=self.project_id,
                     dataset=self.dataset_id,
                     table=OBSERVATION)
-        queries = [
-            observation_tmpl,
-            person_tmpl,
-            primary_map_tmpl,
-        ]
+
+        # a list of PID's that are aian
+        aian_tmpl = self.jinja_env.from_string("""
+        INSERT INTO `{{project}}.{{dataset}}.{{table}}`
+          (person_id)
+        VALUES
+          (500)
+        """).render(
+            project=self.project_id,
+            dataset=self.sandbox_id,
+            #dataset=self.dataset_id,
```

Comment:
Here, you are setting the value of dataset to `self.sandbox_id`.  So, you created the table in one dataset and try to insert data into a table (that doesn't exist) in another dataset.  If you create this in one dataset, that should be the dataset value you use for your insert statement.  These cannot work.

Reply:
Thanks, I chose `f'self.sandbox_id'`